en/frames_plane/reptile_dragon_2.md

```diff
@@ -0,0 +1,391 @@
+With counter rotating props, the airplane can be set up such that it has no [critical motors](https://en.wikipedia.org/wiki/Critical_engine).
+
+With no critical motors, controllability will be maximized if a motor fails.
+The motor direction should be set such that props should turn towards the fuselage on top of the plane.
+In other words, if you look at the left motor with the airplane facing away from you, it should spin clockwise while the right motor should spin counter clockwise.
+
+With the propellers removed, power the airplane up and use the [Actuator](../config/actuators.md) test in QGC to spin up the motors.
+If the left or right motor does not spin in the correct direction, swap two of its ESC leads and check it again.
+Finally, when both motors are spinning the correct directions, use a wrench to attach the propellers.
+
+### Final Checks
+
+Prior to the first flight, a comprehensive preflight must be conducted.
+
+I recommend checking the following items:
+
+- Sensor calibration (QGC)
+  - Mag calibration
+  - Accelerometer calibration
+  - Airspeed calibration
+  - Level horizon calibration
+- Check control surface deflection
+ - Right stick -> Right aileron goes up, left aileron goes down
+ - Left stick -> Left aileron goes up, right aileron goes down
+ - Stick back -> elevator goes up
+ -Stick forward -> elevator goes down
+ - Left rudder -> Rudder goes left
+ - Right rudder -> Rudder goes right
+- Check Px4 inputs (in `stabilized mode`)
+ - Roll right -> Right Aileron goes down
+ - Roll left -> Left aileron goes down
+ - Pitch up -> Elevator goes down
+ - Pitch down -> Elevator goes up
+
+### First Flight
+
+I recommend performing the first takeoff in manual mode.
+Because this airplane has no landing gear, you will either need to throw the airplane yourself, or ideally have a helper throw it.
+When throwing any airplane, throw at a slightly nose up attitude with full throttle.
+
+It's critical to be ready to give aft stick input to prevent the airplane from impacting the ground if it happens to be trimmed nosedown.
+Once the airplane is successfully airborne, cruise up to an altitude of a few hundred feet and switch to [Acro mode](../flight_modes/acro_fw.md).
+If the airplane is well behaved in Acro mode, switch to [Position mode](https://docs.px4.io/main/en/flight_modes/position_fw.html).
+
+Optionally, you can use [Autotuning](../config/autotune.md) to tune the airframe.
+
+## Build Results & Performance
+
+Overall, this build was a success.
+The RD2 flies well in this configuration and has plenty of room onboard for sensors and additional hardware.
+
+### Performance
+
+- Stall speed: 15mph indicated
+- Cruise speed: 35-50mph
+- Endurance: ~40 minutes at 28mph
+
+### Videos & Flight Logs
+
+[Demo Flight log](https://review.px4.io/plot_app?log=6a1a279c-1df8-4736-9f55-70ec16656d1e)
+
+FPV video of flight log:
+
+@[youtube](https://www.youtube.com/watch?v=VqNWwIPWJb0&ab_channel=ChrisSeto)
+
+### Params

```

Comment:
Shouldn't this section go up before first flight? After all, it is untuned at this point?

Reply:
It's tuned in the video, this is a demo flight after all setup is complete.

Reply:
@chris1seto No, I mean "don't you need to load these params to complete the configuration"? - without them the vehicle won't fly.

So my question is "shouldn't these parameters appear before here in the document" because you won't get to first flight without them?